.github/workflows/ephemeral-env.yml

```diff
@@ -4,6 +4,11 @@ on:
   issue_comment:
     types: [created]
 
+# cancel previous workflow jobs for PRs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.issue.number || github.run_id }}
```

Comment:
this is the only edit that's different from the other ones, pointing to `github.event.issue.number` instead of `github.event.pull_request.number`